src/Logger/LogEntry.php

```diff
@@ -226,7 +226,7 @@ public function formatStartTime() {
    * Format end time.
    */
   public function formatEndTime() {
-    return $this->end_time ? t('Previous run finished @ @end_time', array(
+    return $this->end_time ? t('Previous run finished @end_time', array(
```

Comment:
That is probably too much text to show on the table. I think we want to just show a formatted start and end time.